src/index.html

```diff
@@ -1,12 +1,530 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+
+    <header class="header">
+      <a class="logo" href="#HOME">
+        <img src="images/logo.png" alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#APPLE">
+              apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#SAMSUNG">
+              samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">
+              смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link "
+              href="#laptops"
+              data-qa="nav-hover"
+            >
+              ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#devices">
+              гаджеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#skreen">
+              планшеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#pics">
+              фото
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#videos">
+              видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <section class="catalog">
+
+        <div class="card-block catalog__item" data-qa="card">
+
+          <img
+            class="card-block__pic"
+            src="./images/imac.jpeg"
+            alt="imac"
+          >
+
+          <div class="card-block__name">
+            <h2 class="card-block__title">
+              Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </h2>
+          </div>
+
+          <div class="card-block__par">
+            <div class="card-block__parag">
```

Comment:
What tag do we use for text, paragraph?
```suggestion
            <div class="card-block__parag">
```